src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1024,7 +1024,7 @@ internal enum ErrorCode
         ERR_BadAwaitInQuery = 1995,
         ERR_BadAwaitInLock = 1996,
         ERR_TaskRetNoObjectRequired = 1997,
-        WRN_AsyncLacksAwaits = 1998,
+        // WRN_AsyncLacksAwaits = 1998, // Removed in https://github.com/dotnet/roslyn/issues/77001
```

Comment:
nit: the link seems superfluous. We could always git blame if needed to track back